classes/local/report/location_report_builder.php

```diff
@@ -69,8 +72,20 @@ public function build_report() {
         }
 
         $report->add_row('total', $totalcount, $totalsum);
-
+        $filedirrows = $DB->get_records('tool_objectfs_reports', ['reporttype' => 'filedir_size']);
+        if (!empty($filedirrows)) {
+            $report->add_rows($filedirrows);
+            $row = end($filedirrows);
+            $deltacount = $row->objectcount - $totalcount;
+            $deltasize = $row->objectsum - $totalsum;
+            $key = 'deltaa';
+            if ($totalsum > $row->objectsum) {
+                $deltacount = $totalcount - $row->objectcount;
+                $deltasize = $totalsum - $row->objectsum;
+                $key = 'deltab';
+            }
+            $report->add_row($key, $deltacount, $deltasize);
```

Comment:
can you add this row regardless please?

bonus points, add a link here to scheduled task gui to update the stats to the latest

Reply:
updated